uberenv.py

```diff
@@ -331,7 +333,7 @@ def setup_paths_and_dirs(self):
 
 
     def find_spack_pkg_path(self,pkg_name):
-        r,rout = sexe("spack/bin/spack find -p " + pkg_name,ret_output = True)
+        r,rout = sexe("{} find -p ".format(spack_cmd) + pkg_name,ret_output = True)
```

Comment:
I think this would be clearer as:

```python
r,rout = sexe("{0} find -p {1}".format(spack_cmd, pkg_name), ret_output = True) 
```

I.e. format the entire string at once.

Reply:
Also, for consistency, we're adding explicit indexes to the `String.format` lines.
i.e.
Prefer `"{0} find -p {1}".format(...)`  
To `"{} find -p {}".format(...)`

Reply:
Done